{{cookiecutter.repostory_name}}/app/Dockerfile

```diff
@@ -1,19 +1,38 @@
-FROM python:3.6
+{% if cookiecutter.use_alpine_linux == "y" -%}
+ARG BASE_IMAGE=python:3.6-alpine
+{% else -%}
+ARG BASE_IMAGE=python:3.6
+{% endif -%}
 
-COPY ./bin/s6-overlay-amd64.tar.gz /tmp/s6-overlay-amd64.tar.gz
+FROM $BASE_IMAGE AS base-image
 
-RUN tar xvfz /tmp/s6-overlay-amd64.tar.gz -C / \
-    && rm -f /tmp/s6-overlay-amd64.tar.gz {% if cookiecutter.use_celery == "y" %}\
-    && adduser --disabled-password --gecos '' celery{% endif %}
+WORKDIR /root/src/
+ENV PYTHONUNBUFFERED=1
+ENV PATH=/root/.local/bin:$PATH
```

Comment:
do we need those env vars in the first stage?

Reply:
`PYTHONUNBUFFERED=1` no.
but `PATH=/root/.local/bin:$PATH` helps to avoid warnings printed at dependencies installation saying that you install something (`--user`) out of the `PATH`